ansible/playbook.yaml

```diff
@@ -0,0 +1,71 @@
+- name: Update web servers
+  hosts: webservers
+  remote_user: ec2-user
+
+  vars_prompt:
+    - name: trello_api_key
+      prompt: Please input Trello Api Key
+      private: yes
+
+    - name: trello_api_token
+      prompt: Please input Trello Api Token
+      private: yes
+
+  tasks:
+  - name: Install the latest version of git
+    become: yes
+    yum:
+      name: git
+      state: latest
+
+  - name: Install the latest version of python
+    become: yes
+    yum:
+      name: python3
+      state: latest
+
+  - name: Install poetry
+    shell: curl -sSL https://install.python-poetry.org | python3 -
+    args:
+      creates: /home/ec2-user/.local/bin/poetry
+
+  - name: Create directory for the app
+    become: yes
+    shell: mkdir /opt/todoapp
+    args:
+      creates: /opt/todoapp
+
+  - name: Make ec2-user owner of app directory
+    become: yes
+    shell: chown ec2-user /opt/todoapp

```

Comment:
This works well, but my own recommendation would be to lean into the [Ansible file module](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/file_module.html); that both lets us combine these two steps, and (hopefully) can handle more edge cases than we might initially consider!
```yml
- name: Create a directory for the app
  ansible.builtin.file:
    path: /opt/todoapp
    state: directory
    owner: ec2-user
  become: yes
```